docs/docs/cmd/aad/group/group-list.md

```diff
@@ -0,0 +1,21 @@
+# aad groupsetting list
+
+Lists Azure AD groups
+
+## Usage
+
+```sh
+m365 aad group list [options]
+```
+
+## Options
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+List all groups in the tenant
```

Comment:
Let's clarify this so that it's explicit that this command is about retrieving groups defined in Azure Active Directory. _groups in the tenant_ is ambiguous with Azure AD and O365 groups.